uwblueprint/jump-math
# backend: delete classes associated with teacher on delete teacher

## Notion ticket link
[Delete classes associated with teacher on deleteTeacher](https://www.notion.so/uwblueprintexecs/Delete-classes-associated-with-teacher-on-deleteTeacher-80e2b997f245470ab29c4cbe584eef55?pvs=4)



## Implementation description
* Added deleting classes to `deleteUserById` and `deleteUserByEmail` mutation in `backend/graphql/resolvers/userResolvers.ts`



## Steps to test
1. In GraphQL playground, add a user with class id's array (choose class ids from MongoDB atlas)
2. In the plaground, delete the user by email 
3. Check MongoDB atlas to see if the proper user is deleted along with the classes  


## Checklist
